modules/architecture-machine-roles.adoc

```diff
@@ -44,15 +44,20 @@ Services that fall under the Kubernetes category on the master include the Kuber
 |The Kubernetes API server validates and configures the data for pods, services,
 and replication controllers. It also provides a focal point for cluster’s shared
 state.
+
 |etcd
 |etcd stores the persistent master state while other components watch etcd for
 changes to bring themselves into the specified state.
 //etcd can be optionally configured for high availability, typically deployed with 2n+1 peer services.
+
 |Controller Manager Server
 |The Controller Manager Server watches etcd for changes to objects such as
 replication, namespace, and serviceaccount controller objects, and then uses the
 API to enforce the specified state. Several such processes create a cluster with
 one active leader at a time.
+
+|Kubernetes scheduler
+|The Kubernetes scheduler watches for newly created pods and selects the best node for the pod to run on.
```

Comment:
Same preposition de-clutter suggestion at the end of this sentence, "and selects the best node to run the pod."